src/Polly.Extensions/DependencyInjection/PollyServiceCollectionExtensions.cs

```diff
@@ -16,6 +17,79 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// </summary>
 public static class PollyServiceCollectionExtensions
 {
+    /// <summary>
+    /// Adds a generic resilience strategy to service collection.
+    /// </summary>
+    /// <typeparam name="TKey">The type of the key used to identify the resilience strategy.</typeparam>
+    /// <typeparam name="TResult">The type of result that the resilience strategy handles.</typeparam>
+    /// <param name="services">The <see cref="IServiceCollection"/> to add the resilience strategy to.</param>
+    /// <param name="key">The key used to identify the resilience strategy.</param>
+    /// <param name="configure">An action that configures the resilience strategy.</param>
+    /// <returns>The updated <see cref="IServiceCollection"/> with the registered resilience strategy.</returns>
+    /// <exception cref="InvalidOperationException">Thrown if the resilience strategy builder with the provided key has already been added to the registry.</exception>
+    /// <remarks>
+    /// You can retrieve the registered strategy by resolving the <see cref="ResilienceStrategyProvider{TKey}"/> class from the dependency injection container.
+    /// <para>
+    /// This call enables the telemetry for the registered resilience strategy.
```

Comment:
Is this comment correct?

Reply:
Yes.

Reply:
I don't quite follow how it turns on telemetry - don't you need to call `EnableTelemetry()` to do that?